website/docs/topics/feature-flags/best-practices-using-feature-flags-at-scale.md

```diff
@@ -0,0 +1,342 @@
+
+- [Organizing your team and internal processes to support feature flags](#organizing-your-team-and-internal-processes-to-support-feature-flags)
+  - [1. Align feature flags with organizational and application design.](#1-align-feature-flags-with-organizational-and-application-design)
+  - [2. Make flags searchable globally.](#2-make-flags-searchable-globally)
+  - [3. Design for flag permissions to change over time.](#3-design-for-flag-permissions-to-change-over-time)
+  - [4. Get flag management permissions right. And audit everything.](#4-get-flag-management-permissions-right-and-audit-everything)
+  - [5. Base flag permissions on global permissions.](#5-base-flag-permissions-on-global-permissions)
+  - [6. Implement flag approval workflows early.](#6-implement-flag-approval-workflows-early)
+- [Instrumenting your code and managing flag lifecycle](#instrumenting-your-code-and-managing-flag-lifecycle)
+  - [7. Define flags in your code at the highest level of abstraction.](#7-define-flags-in-your-code-at-the-highest-level-of-abstraction)
+  - [8. Evaluate a feature flag only once.](#8-evaluate-a-feature-flag-only-once)
+  - [9. Continually pay down feature flag tech debt.](#9-continually-pay-down-feature-flag-tech-debt)
+  - [10. Leverage flag lifecycles to optimize your development](#10-leverage-flag-lifecycles-to-optimize-your-development)
+- [Common pitfalls to avoid when implementing flags at scale](#common-pitfalls-to-avoid-when-implementing-flags-at-scale)
+  - [11. Avoid feature flag parent-child dependencies.](#11-avoid-feature-flag-parent-child-dependencies)
+  - [12. Don’t use flags to manage configuration.](#12-dont-use-flags-to-manage-configuration)
+  - [13. Never reuse feature flag names.](#13-never-reuse-feature-flag-names)
+  - [14. Avoid giant feature flag targeting lists.](#14-avoid-giant-feature-flag-targeting-lists)
+  - [15. Do not define business logic using feature flags.](#15-do-not-define-business-logic-using-feature-flags)
+- [Staying sane while managing feature flags at scale](#staying-sane-while-managing-feature-flags-at-scale)
+
+## Organizing your team and internal processes to support feature flags
+
+Before you add your first feature flag, you need to think about how organizational structure and processes that live outside of code will affect your feature flag deployments.
+
+### 1. Align feature flags with organizational and application design.
+
+> Any organization that designs a system will produce a design whose structure is a copy of the organization's communication structure.
+
+– Melvin Conway
+
+<Figure img="/img/feature-flag-organization-methods.png" caption="In situtations where predefined teams own a feature from start to finish, it makes sense to create a project for each team. In situtations where you have dynamic teams that form around feature work, you might instead want to create smaller projects for each feature, only giving access to the people who work on that feature."/>
```

Comment:
```suggestion
<Figure img="/img/feature-flag-orginization-methods.png" caption="In situtations where predefined teams own a feature from start to finish, it makes sense to create a project for each team. In situtations where you have dynamic teams that form around feature work, you might instead want to create smaller projects for each feature, only giving access to the people who work on that feature."/>
```

Reply:
@ferrantim You've put a typo in the file name of this one, so the image wasn't loading correctly. I can't easily change the file name, but I can update the reference. Not super important, but I suggest you go and update the file name and the reference later.